FAQ.md

```diff
@@ -185,12 +185,12 @@ together in a number of configurations:
 
 ## Why not just use LLVM bitcode as a binary format?
 
-The [LLVM](https://llvm.org/) compiler infrastructure has a lot to recommend it:
+The [LLVM](https://llvm.org/) compiler infrastructure has a lot to recommend:
```

Comment:
The original text here was correct. It was using the definition of "recommend"  that appears fourth here: https://www.dictionary.com/browse/recommend. This could still be clarified by rewording it to something like "has attractive qualities," though.,

Reply:
Huh I never knew that, it sounded wrong to me but as per the dictionary it's right. Rephrased as per your suggestion.